app/controllers/stages_controller.rb

```diff
@@ -145,4 +145,11 @@ def find_project
   def find_stage
     @stage = @project.stages.find_by_param!(params[:id])
   end
+
+  def update_deploy_groups
+    # http://stackoverflow.com/questions/8929230/why-is-the-first-element-always-blank-in-my-rails-multi-select-using-an-embedde
+    deploy_group_ids = params[:stage].fetch(:deploy_groups, []).select { |id| !id.empty? }
```

Comment:
if you mean why couldn't the '@stage.update_attributes(stage_params)' handle the array of IDs, it couldn't handle the array of IDs with or without the blank element that had to be stripped.
Do you know of another way of doing this?